<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDds</class>
 <widget class="QWidget" name="FDds">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список суточных сеансов ДДС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Тип файла</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="cb_kvi">
             <property name="text">
              <string>КВИ</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_kvno">
             <property name="text">
              <string>КВНО</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Уровень обработки</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="rb_all">
             <property name="text">
              <string>Все</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_obr">
             <property name="text">
              <string>Обработанные</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_err">
             <property name="text">
              <string>Ошибки</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_nobr">
             <property name="text">
              <string>Необработанные</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>№ КА:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbNka">
         <item>
          <property name="text">
           <string>все</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>126</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>127</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Refresh">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_List">
        <property name="text">
         <string>Просмотр</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Text.png</normaloff>:/res/Text.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Stat">
        <property name="text">
         <string>Статистика</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Stock graph.png</normaloff>:/res/Stock graph.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnProt">
        <property name="text">
         <string>Протокол</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/page_update.png</normaloff>:/res/1/page_update.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Konv">
        <property name="text">
         <string>Конвертировать</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/default_value.png</normaloff>:/res/1/default_value.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Vigr">
        <property name="text">
         <string>Выгрузить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Del">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/bin.png</normaloff>:/res/1/bin.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Graf">
        <property name="text">
         <string>График</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/1/chart_column_horizont.png</normaloff>:/res/1/chart_column_horizont.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>889</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
